tests/quantizers_infrastructure_tests/pytorch_tests/test_inferable_quantizers/test_activations_inferable_quantizer.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
consider refactoring the tests to verify the quantization executes the desired quantization function (written explicitly here in numpy)m instead of checking implicit quantization output values like clipping range